tests/integration/test_claim_bribes.py

```diff
@@ -244,14 +255,28 @@ def test_claim_bribes_with_redirection(
     # Confirm GNO event
     assert event[1]["destination"] == gno_recepient
     assert event[1]["token"] == gno
-    assert event[1]["amount"] == gno_recepient_balance_before + gno_amount - expected_gno_fee_amount
+    assert (
+        event[1]["amount"]
+        == gno_recepient_balance_before + gno_amount - expected_gno_fee_amount
```

Comment:
a typo `recipient` ?

Reply:
Good catch fixing on the test but leaving the strat untouched to reduce the changes